.env.development.local

```
@@ -0,0 +1,4 @@
+#VITE_CLIENT_GUID=f51060e1c38446f0bacdf283390c37e8
+#VITE_API_URL=https://searchapi-dev.hawksearch.net
+VITE_CLIENT_GUID=3953cf87e0aa4a4eb9496a24d5a0fa84
+VITE_API_URL=https://essearchapi-na.hawksearch.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_API_URL key should go before the VITE_CLIENT_GUID key

(UnorderedKey)